tools/buildsteps/windows/BuildSetup.bat

```diff
@@ -197,7 +197,11 @@ set WORKSPACE=%base_dir%\kodi-build.%TARGET_PLATFORM%
   IF EXIST %APP_SETUPFILE% del %APP_SETUPFILE% > NUL
 
   rem determine if current system is 32 or 64 bits
-  IF %PROCESSOR_ARCHITECTURE% == AMD64 (
+  SET HOST_BITS=32
+  IF %PROCESSOR_ARCHITECTURE% == AMD64 SET HOST_BITS=64
+  IF %PROCESSOR_ARCHITECTURE% == ARM64 SET HOST_BITS=64
```

Comment:
Is the duplicate line intentional?

Reply:
Not duplicate, one is arm64, other is amd64